### ISSUE TITLE
Fix way we model 'measurements' in factorvalues.

### ISSUE THREAD

**Issue by ppavlidis** [PavlidisLab/Gemma#941]
There is a problem with how these are currently handled in the Statement framework, and especially in serialization, but this was an issue already.

We need to express that the value (a number) is the measurement of "Age" or whatever. Currently that is only held in the ExperimentalFactor?

**Comment by arteymix** [comment]
I see, so for a measurement, the category of the factor should be included in the serialization. It shouldn't be too difficult to make that adjustment.

**Comment by ppavlidis** [comment]
Probably not just the category, but the description of the ExperimentalFactor. 

I don't have examples handy, but we might have "Phenotype" as the category, but the actual measurement might be age, BMI, blood pressure etc.

The "existing problem" was that we don't have an ontology term slot for "blood pressure" in such cases (for example), but it's not a big deal.

**Comment by arteymix** [comment]
We can always attach a characteristic to the measurement if we need to be more precise. We could handle measurement with a single characteristic as describing the nature of what is being measured.

**Comment by ppavlidis** [comment]
Yes, that would be the best solution. Right now, when a curator clicks on a continuous factor, the factorvalue pane is greyed out. We should instead allow a characteristic to be added there, but this will require curation in addition to a data model change. The change to the current serialization would be a workaround for now.

**Comment by ppavlidis** [comment]
I don't think that really fixes it? If you just use the Category, it will often not be descriptive or could be the same as a different factor.

For example, the category for pH might be "phenotype", but we need to see "pH:6.9". So this fix at papers over the problem, but it needs more work, if I'm understanding correctly. "Age" is one of the few cases where the category is very specific .

**Comment by arteymix** [comment]
I agree, the category is not sufficient.

It would be very cumbersome to track the annotation in each factor value for measurements because we would have to duplicate characteristics when we assign a value to a sample.

I'm thinking of an approach where factor values would inherit the annotations of their experimental factor. We would have to repurpose the existing annotations collection.

**Comment by ppavlidis** [comment]
Gemma 2 disposition: Fixed (6471fe9476 — include factor category when serializing a measurement)
